srcDoc/release_notes.rst

```diff
@@ -54,6 +54,13 @@ What's New in Master
 What's Fixed in Master
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Error using Micro with SHMEM when the test finalize
```

Comment:
Error finalizing the application when using `SHMEM` for `RTI Connext DDS Micro` (#234)